hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpDeleteCommandParser.java

```diff
@@ -29,6 +30,11 @@
 
     @Override
     public TextCommand parser(TextDecoder decoder, String cmd, int space) {
+        throw new UnsupportedOperationException("not implemented");
```

Comment:
in `HttpPostCommandParser` such method throws
`throw new IllegalStateException();`
Should POST and DELETE be different?